lib/geokit-rails/acts_as_mappable.rb

```diff
@@ -137,16 +137,16 @@ def by_distance(options = {})
         bounds  = extract_bounds_from_options(options)
         distance_column_name = distance_sql(origin, units, formula)
         #geo_scope(options).order("#{distance_column_name} asc")
-        order("#{distance_column_name} asc")
+        order("#{distance_column_name} #{options[:reverse] ? 'DESC' : 'ASC'}")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.